exercises/timesTable13.js

```diff
@@ -8,10 +8,18 @@
  *   13 times 3 is 39
  *   13 times 4 is 52
  */
+//I couldnt figure out how to print out all the multiples under the parameter
+//Im assuming I need an array to store the multiples as they run through
+//I only got it to print the input given.
 function timesTable13(n) {
-  // This is your job. :)
+  for (i = 1; i <= 1000; i++) {
+    if (n % 13 === 0) {
+      console.log('13 times', n, 'is', 13 * n)
```

Comment:
As in the above comment, you'd use `i` here instead of `n`.

Reply:
With all those changes, you'd have a function that prints out every multiple of 13 smaller than `n`!

Reply:
But to print out the first `n` multiples of 13, you'd do:

```javascript
function timesTable13(n) {
  for (let i = 1; i <= n; i++) {
    console.log('13 times', i, 'is', 13 * i);
  }
}
```